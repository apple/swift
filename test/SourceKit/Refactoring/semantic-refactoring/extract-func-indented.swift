func foo2() -> Int {
  func foo3() -> Int {
    var a = 3
    a = a + 1
    return 1
  }
}

// RUN: %empty-directory(%t.result)
// RUN: %sourcekitd-test -req=extract-func -pos=3:1 -end-pos 5:13 -name new_name %s -- %s > %t.result/extract-func-idented.swift.expected
// RUN: diff -u %S/extract-func-idented.swift.expected %t.result/extract-func-idented.swift.expected

// FIXME: Fails on linux with assertion: "!GlibcModuleMapPath.empty()"" failed
// REQUIRES-ANY: OS=macosx
