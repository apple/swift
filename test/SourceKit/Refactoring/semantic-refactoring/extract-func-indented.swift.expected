source.edit.kind.active:
  2:3-2:3 "func new_name() -> Int {
    var a = 3
    a = a + 1
    return 1
  }

  "
  <note>source.refactoring.range.kind.basename 1:6-1:14 "new_name"</note>
source.edit.kind.active:
  3:5-5:13 "return new_name()"
  <note>source.refactoring.range.kind.basename 1:8-1:16 "new_name"</note>
